src/pages/ShoppingCard.js

```diff
@@ -135,3 +137,31 @@ const Gesamtpreis = styled.div`
     color: red;
   }
 `;
+export const NextButton = styled.button`
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  border-radius: 10px;
+
+  &:hover {
+    background-color: aliceblue;
+    cursor: pointer;
```

Comment:
cursor: pointer siehe oben

Reply:
here is necessary

Reply:
Ja, cursor: pointer is richtig, aber an der falschen Stelle. Es gehört nicht in hover, sondern in den basic style.